packages/block-library/src/social-link/edit.js

```diff
@@ -81,8 +81,7 @@ const SocialLinkURLPopover = ( {
 					/>
 				</div>
 				<Button
-					// TODO: Switch to `true` (40px size) if possible
-					__next40pxDefaultSize={ false }
+					__next40pxDefaultSize
```

Comment:
This can be seen, while you add `core/social-link` block, and add the link. You can check for the `apply` button.

| Before | After     |
| :-------- | :------- |
| <img width="1331" alt="social-link-before" src="https://github.com/user-attachments/assets/a61a1caf-12dc-48bb-8071-f77c9fd07707"> | <img width="1331" alt="social-link-after" src="https://github.com/user-attachments/assets/3d3f074a-211c-4654-accb-dd916bcf28b0"> |

Reply:
We should update the `URLInput` to a `40px` height too — @mirka , do you know if that's feasible as of now? Or should we keep the TODO comment and revisit later?

Reply:
Mmm, this is kind of dependent on #64709, so I guess we can revert to TODO for now. I'll do something about it if the Button migrations get completed before the URLInput issue is addressed.

Reply:
Sure, I would update this to TODO state.